core/modules/installer/installer.manager.inc

```diff
@@ -364,6 +364,13 @@ function installer_manager_download_batch_finished($success, $results) {
     // but just in case, we have to tell them something.
     backdrop_set_message(t('Fatal error trying to download.'), 'error');
   }
+
+  // Now that the update is done, we can put the site back online if it was
+  // previously in maintenance mode.
+  if (isset($_SESSION['maintenance_mode']) && $_SESSION['maintenance_mode'] == FALSE) {
```

Comment:
Can we please change this to `$_SESSION['previous_maintenance_mode']`? (see my next comment)

Reply:
This is used all over the place, I don't want to break something because we wanted to change an array key.